web/serve_locally.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
```

Comment:
I gave an attempt at using some modern workflow for web dev, but got overwhelmed. So for the moment, no bundling and stuff needed; the .html just pulls in all JS and CSS dependencies from a CDN. And to just run locally, this is my hack. If there's an npm / yarn / whatever tutorial that's easy to follow and would just let me bring in a few dependencies and not all of React, happy to follow that!

Reply:
its simple and it works.. I have not complaints.

I'll look into something structural, but only after the talk